scripts/libsimple_crontab

```diff
@@ -0,0 +1,23 @@
+# /etc/cron.d/circulation: Library Simplified Circulation Manager crontab
+# You don't have to run the `crontab' command to install the new
+# version when you edit this file in /etc/cron.d. Files in this directory
+# also have username fields, similar to the systemwide /etc/crontab.
+
+SHELL=/bin/sh
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+HOME=/var/www/circulation/bin
+
+# m h dom mon dow user command
+*/15 * * * * root ./run axis_monitor >> /var/log/cron.log 2>&1
```

Comment:
Is everything really running as root? I don't know if that's a problem but it's not what I expected.